pages/ledger/index.vue

```diff
@@ -58,7 +58,7 @@
       <br />
       <br />
     </div>
-    <div v-if="!isWindows && !isLinux" class="session-main">
+    <div v-if="!isChromiumBrowser" class="session-main">
```

Comment:
ok cool - i see the fix. what i was trying to say is that if this is an error message, why not just show it in {{ error }}? do we need a special message just for this one case?

Reply:
maybe we make this a computed property where it sets the `error` if it's a chromium browser

Reply:
I think there are too many possible errors, Jordan. Depends if it is IE, Opera, Firefox... there are just too many browsers out there. This is better, trust me.

In Firefox the error is "there is no `window.usb`", in Safari is another... Too many possibilities

Reply:
nevermind. all good.